tests/test_biophysical/test_animal/test_cow_calf/test_beef_ration.py

```
@@ -0,0 +1,496 @@
+    with pytest.raises(ValueError, match="non-negative|sum to 100"):
+        RationManager.set_ration_feeds(bad_config)
+    assert RationManager.beef_lactating_pasture_ration is SENTINEL_LACTATING
+    assert RationManager.beef_dry_gestating_ration is SENTINEL_DRY
+    assert RationManager.beef_creep_feed_ration is SENTINEL_CREEP
+    assert RationManager.beef_replacement_heifer_ration is SENTINEL_HEIFER
+
+
+# ── Task 6.2: Seasonal ration selection ───────────────────────────────────────
+
+
+@pytest.fixture()
```

Ruff (0.15.20)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)